src/parser/index.ts

```diff
@@ -214,3 +228,47 @@ function extractTokens(ctx: Context) {
     return /^[^\w$]$/iu.test(c);
   }
 }
+
+function parseStyleAst(
+  styleElement: SvelteStyleElement,
+  ctx: Context
+): Root | null {
+  if (!styleElement.endTag) {
+    return null;
+  }
+  let lang = "css";
+  for (const attribute of styleElement.startTag.attributes) {
+    if (
+      attribute.type === "SvelteAttribute" &&
+      attribute.key.name === "lang" &&
+      attribute.value.length > 0 &&
+      attribute.value[0].type === "SvelteLiteral"
+    ) {
+      lang = attribute.value[0].value;
+    }
+  }
+  let parseFn: Parser<Root> | undefined = postcss.parse;
+  switch (lang) {
+    case "css":
+      parseFn = postcss.parse;
+      break;
+    case "scss":
+      parseFn = SCSSparse;
+      break;
+    default:
+      console.warn(`Unknown <style> block language "${lang}".`);
```

Comment:
@ota-meshi What do you think is the right thing to do here? Probably not using the console?